src/proc/colorizer.cpp

```diff
@@ -245,10 +245,13 @@ namespace librealsense
         if (!frame || frame.is<rs2::frameset>())
             return false;
 
-        if (frame.get_profile().stream_type() != RS2_STREAM_DEPTH)
-            return false;
+        std::set<rs2_format> supported_formats = {RS2_FORMAT_Z16, RS2_FORMAT_DISPARITY32};
```

Comment:
Sorry , I meant building this set as static const or at least only once and not on every frame.